### PR TITLE
Homework lesson 1: Duong Van Truong

### PR DESCRIPTION
**Pull request by tadashii1417**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (trongtai37) [inline .gitignore]: I think you should not push this kind of file.

File: homework/Truong_Duong/lesson_1/202_Happy_Number.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+class Solution:
+    def isHappy(self, n: int) -> bool:
+        mset = {}
+
+        while n not in mset:
+            mset[n] = 1
+            currentSum = 0
+
+            while n != 0:
```
Comment (quanghoang33) [inline homework/Truong_Duong/lesson_1/202_Happy_Number.py:8]: consider move this logic to a separated function

File: homework/Truong_Duong/lesson_1/304_Range_sum_querry_2D.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Solution for https://leetcode.com/problems/range-sum-query-2d-immutable/
+
+class NumMatrix:
+    def getVal(self, i, j):
```
Comment (quanghoang33) [inline homework/Truong_Duong/lesson_1/304_Range_sum_querry_2D.py:4]: nice

File: homework/Truong_Duong/lesson_1/202_Happy_Number.py (hunk 2 above)
Comment (tadashii1417) [reply homework/Truong_Duong/lesson_1/202_Happy_Number.py:8]: Thank you, I fixed it 🙇‍♂️

**Pull request merged**